yarn/Dockerfile.agent

```diff
@@ -0,0 +1,29 @@
+FROM ubuntu:14.04
+MAINTAINER Klaus Ma <madaxa@cn.ibm.com>
+
+RUN apt-get update && apt-get -y install openjdk-7-jdk openssh-server
+
+COPY hadoop-2.7.2.tar.gz /opt/
+
+RUN cd /opt/ && tar zxvf hadoop-2.7.2.tar.gz && rm hadoop-2.7.2.tar.gz
+
+COPY ./agent_bootstrap.sh /opt/bootstrap.sh
```

Comment:
each of these COPY's produce a layer.  Can we get them in one COPY.  Pattern I usually follow is copy the files in then use RUN to move them to the correct place.  This docker is creating a ton of layers.